sources/Application/Utils/HelpLegend.h

```diff
@@ -0,0 +1,98 @@
+#include <cstdio>
+#include <cstring>
+
+static inline const char (*getHelpLegend(const char *fx))[30] {
+  static char result[3][30];
+  strcpy(result[0], fx);
+  strcpy(result[2], "bb at speed aa");
```

Comment:
I think this will still use more ram than is necessary because you are doing string copies at runtime into the statically allocated `result` which takes up 3x30 bytes for the entire time the firmware is running.

What I would suggest instead is to have 3 char pointers in an array: `static char* result[3];` and then you can just point the pointers [sic] at the string literals addresses, which the compiler should have taken care of putting into flash, eg
```c
 if (strcmp(fx, "KIL") == 0) {
    result[0] = &", KILl: stop playing after";
    tesult[1] = &" aa ticks"; 
...etc
```  
so then we are only using up 3 bytes of extra ram to implement this feature.